content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -180,19 +181,19 @@ Client: BYE
 Closing connection
 ```
 
-### Client side Programming
-# Establish a Socket Connection
-This is an instance of a socket class and can be obtained in two ways: 
+### Client-side Programming
+#### Establish a Socket Connection
+A Socket instance can be obtained in two ways: 
     
-1. On the server side it can be used as a return value of the accept ( ) method.
+1. On the server side it can be used as a return value of the accept method.
 2. The builder of socket objects accepts the hostname of the server on which the server operates and the server's port number.
 
 ```java
 socket = new Socket("localhost", 5000);
 ```
 
 localhost - is a domain name like google.com, localhost takes you to your computer. It is resolved to IP address 127.0.0.1 .
```

Comment:
Change:

>localhost - is a domain name like google.com, localhost takes you to your computer. It is resolved to IP address 127.0.0.1 .
The second argument is a port number.
Two computers communicate through sockets.

To:

>The first argument: `"localhost"` is a domain name like google.com. It is a representation of your computer which a machine resolves to IP address 127.0.0.1. Additionally, the second argument is the port number we chose.

Removed passive voice, removed an unnecessary space, improved sentence flow. Also: I felt the last sentence: **Two computers communicate through sockets.** was unnecessary since you already mentioned it so I removed it.